debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnector.java

```diff
@@ -50,11 +54,47 @@ public void start(Map<String, String> props) {
 
     @Override
     public List<Map<String, String>> taskConfigs(int maxTasks) {
-        if (maxTasks > 1) {
-            throw new IllegalArgumentException("Only a single connector task may be started");
-        }
+        if (!properties.containsKey(DATABASE_NAMES.name())) {
+            if (maxTasks > 1) {
+                throw new IllegalArgumentException("Only a single connector task may be started");
+            }
 
-        return Collections.singletonList(properties);
+            return Collections.singletonList(properties);
+        }
+        else {
+            try {
+                // Parse the database names property
+                String[] dbNames = properties.get(DATABASE_NAMES.name()).split(",");
+
+                // Initialize the database list for each task
+                List<List<String>> taskDatabases = new ArrayList<>();
+                for (int i = 0; i < maxTasks; i++) {
+                    taskDatabases.add(new ArrayList<>());
+                }
+
+                // Add each database to a task list via round robin.
+                for (int dbNamesIndex = 0; dbNamesIndex < dbNames.length; dbNamesIndex++) {
+                    int taskIndex = dbNamesIndex % maxTasks;
+                    taskDatabases.get(taskIndex).add(dbNames[dbNamesIndex]);
+                }
+
+                // Create a task config for each task, assigning each a list of database names.
+                List<Map<String, String>> taskConfigs = new ArrayList<>();
+                for (int taskIndex = 0; taskIndex < maxTasks; taskIndex++) {
+                    String databases = String.join(",", taskDatabases.get(taskIndex));
+                    Map<String, String> taskProperties = new HashMap<>(properties);
+                    taskProperties.put(DATABASE_NAMES.name(), databases);
+                    taskProperties.put(DATABASE_NAME.name(),
+                            taskDatabases.get(taskIndex).get(0));
```

Comment:
Eventually, a singular database name shouldn't be required if a list is provided. Right now, the rest of the code still depends on it, so we cannot get rid of it right now. Probably worth filing a separate case.